tests/spec/core/utils-spec.js

```diff
@@ -363,7 +363,11 @@ describe("Core - Utils", () => {
   });
 
   describe("joinAnd", () => {
-    it("joins with proper commas and 'and'", () => {
+    it("joins with proper commas and 'and' when Intl.ListFormat is not available", () => {
```

Comment:
I don’t think we need these modifications, as the use of the intl API is an implementation detail.

Reply:
I think this is essential as the tests run in a browsers, and only one of the two code blocks will be executed, depending on whether or not the browser has ```Intl.ListFormat```
More clearly, if we do away with the tests then only our original code will be tested, and once chrome/firefox incorporate ```Intl.ListFormat```, only the newer code will be tested.

Reply:
I understand the rationale, but then it’s a matter of testing in multiple browsers (what Karma does) of which at least one contains the new API. The tests should primarily test the expected output, otherwise it becomes increasingly difficult to refactor the code. Imagine tomorrow a new intl API is released that was better/faster - it would mean these tests would potentially fail even though they produce the correct output.

Reply:
Okay. Makes sense. Thank you so much for the clarification!

Reply:
No problem. Thanks for hearing me out. Appreciate that you went and tested the behaviour nonetheless!